README.md

```diff
@@ -23,6 +23,8 @@ docker run -p 443:443 -e LDAP_URI="ldap://ldap.example.com/dc=example,dc=com?uid
 |`DHPARAM_PEM`|yes|| Apache's SSL/TLS PEM DHParameter."
 |`CUSTOM_APACHE_CONFIG`|no||Specifies custom parameters to be appended to the apache virtualhost configuration.|
 |`LDAP_URI`|yes||Specifies the URI of the LDAP server, as documented [here](<https://httpd.apache.org/docs/2.4/mod/mod_authnz_ldap.html#authldapurl> "Apache docs").|
+|`LDAP_BIND_DN`|no||Specifies the DN used for search, as documented [here](<https://httpd.apache.org/docs/2.4/mod/mod_authnz_ldap.html#authldapbinddn> "Apache docs").|
```

Comment:
There should be a note about the exclusivity between the `LDAP_BIND_DN` and `LDAP_BIND_USER_PATTERN` parameters